pyproject.toml

```diff
@@ -10,6 +10,8 @@ Flask = "^2.0.3"
 python-dotenv = "^0.14.0"
 requests = "^2.28.1"
 gunicorn = "^20.1.0"
+pymongo = "^4.3.3"
+mongomock = "^4.1.2"
```

Comment:
We could shift this into the dev-dependencies below if we want to, to reduce the attack surface & image size in our production instance